buildah.go

```diff
@@ -19,9 +19,17 @@ const (
 	// identify working containers.
 	Package = "buildah"
 	// Version for the Package
-	Version       = "0.1"
+	Version = "0.1"
+	// The value we use to identify what type of information, currently a
+	// serialized Builder structure, we are using as per-container state.
+	// This should only be changed when we make incompatible changes to
+	// that data structure, as it's used to distinguish containers which
+	// are "ours" from ones that aren't.
 	containerType = Package + " 0.0.1"
```

Comment:
If you get a chance to add verbiage later about the ociv1 spec here, I think it would make it clear, thx.